docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '3'
 
 services:
+    reporter:
```

Comment:
https://github.com/elifesciences/elife-xpub-formula/blob/master/salt/elife-xpub/init.sls#L155 should cover this :+1: